### ISSUE TITLE
Feature Request: Notes and search

### ISSUE THREAD

**Issue by sprnza** [climech/grit#6]
Hi!
Thanks for the great tool.
It seems like it has big potential. Would be great to have ability to add notes to tasks and search through them.

**Comment by Matt-Deacalion** [comment]
Yes, metadata in general would be useful (start time, end time etc).
